docs/HowTo/Voluntary-Exit.md

```diff
@@ -5,11 +5,17 @@ title: Voluntarily exit a validaor
 # Voluntarily exit a validator
 
 A voluntary exit is when a validator chooses to stop performing its duties, and exits the beacon
-chain.
+chain permanently.
+A validator cannot rejoin the network once it voluntarily exits.
+Its funds will be frozen until the withdrawal process is implemented.
```

Comment:
maybe keep this line and remove [line 21](https://github.com/ConsenSys/doc.teku/pull/285/files#diff-5ef2afe21263a4e5717a38cf0b3dc59983694628e431615bd1061330fd085884R21) or write a mix of both, but here it's a bit redundant to me.